### PR TITLE
purge da messages, *i linted okay?!*

### PR DESCRIPTION
**Pull request by hypergamer80**
APPROVE THIS

### CODE REVIEW

File: bot/exts/purge.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    """Cog for purge command"""
+    
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+    
+    
+    @commands.command(aliases=["yeetmsg"])
+    @commands.has_role("Cat Devs")
+    async def purge(self, ctx: commands.Context, limit: int) -> None:
+        await context.channel.purge(limit=int(limit))
```
Comment (Kronifer) [inline bot/exts/purge.py:14]: maybe add a message confirming deletion

**Review by Kronifer — CHANGES_REQUESTED** [review]
Just one thing i think

**Comment by dannynotsmart** [comment]
make a limit on how many messages to delete, because too much can get your bot terminated. also after send a message saying what the bot has done, and who has purged.

File: bot/exts/purge.py (hunk 1 above)
Comment (hypergamer80) [reply bot/exts/purge.py]: did it, resolved.

**Comment by ShakyaMajumdar** [comment]
We can get rid of the `__init__` since you're not using the `bot` attribute anywhere

**Comment by hypergamer80** [comment]
no @ShakyaMajumdar, im using it at the bottom, the setup. we need the init

**Review by dannynotsmart — APPROVED** [review]
ok, everything looks good. I'd suggest putting the max messages at 100 (to be safe from discord api ban), but for now everything looks good.

**Comment by ShakyaMajumdar** [comment]
@hypergamer80 the `bot` in the `setup` is the function parameter, not the instance attribute

**Comment by hypergamer80** [comment]
@ShakyaMajumdar even so, i've seen a lot of code that use this. you're sure it's okay to remove?

**Review by Kronifer — CHANGES_REQUESTED** [review]
Move purge.py to bot/exts/moderation

**Comment by hypergamer80** [comment]
@Kronifer how do i move files?

**Comment by hypergamer80** [comment]
@Kronifer all of your suggestions have been implemented.

**Review by AbooMinister25 — APPROVED** [review]
Everything seems good so far.

**Comment by hypergamer80** [comment]
thanks!

**Comment by Kronifer** [comment]
Merged

**Review by Kronifer — APPROVED** [review]
Lgtm

**Pull request merged**